### ISSUE TITLE
Feature request: integrate with fuzzy finders

### ISSUE THREAD

**Issue by infokiller** [rcaloras/bashhub-client#38]
Example fuzzy finder: https://github.com/junegunn/fzf

IMHO, this is often much easier to use for searching history entries.

**Comment by simeg** [comment]
@infokiller You can downlod `fzf` and just pipe it to achieve the effect you're looking for.

```sh
bh | fzf
```

You could also make an alias function for this. Put the function in `.bashrc` or if you're using zsh `.zshrc`.

```sh
function my_func() {
  bh | fzf
}
```

(You will have to reload this configuration to make it be picked up, `source .bashrc/.zshrc`)

This will make `my_func` be globally available in your terminal.

**Comment by infokiller** [comment]
Thanks @simeg !
Your suggestion will work, but I would also like to get other info about the command such as when it was run.

**Comment by simeg** [comment]
@infokiller Yeah that sounds useful. So you want to be able to fuzzy find commands based on _when_ you entered them?

**Comment by infokiller** [comment]
Yes, my current custom fzf based history selection looks like this:

```
  2017-11-26 23:27:49 | tmux --version
  2017-11-26 23:27:51 | tmux -V
> 2017-11-26 23:27:56 | df -h .
  15596/15596 -S
>
```

It would be great to be able to reproduce this experience with bashhub-client.

**Comment by simeg** [comment]
Sounds useful indeed. It also sounds like something out of scope from this issue. Maybe create a new one? See my PR or the docs for how to integrate with fzf as bashhub works now.

**Comment by infokiller** [comment]
Sounds good, will create a new one, thanks!

**Comment by infokiller** [comment]
Closed in favor of #55
